docusaurus/observability-best-practices/README.md

```diff
@@ -0,0 +1,57 @@
+# Observability Best Practices
+
+## Welcome
+
+This is the source for the [AWS Observability Best Practices site](https://aws-observability.github.io/observability-best-practices/). Everyone is welcome to contribute here, not just AWS employees!
+
+## How to run/develop this site
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn [or] npm
+```
+
+### Local Development
+
+```
+$ yarn start [or] npm run start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```
+$ yarn build [or] npm run build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+Using SSH:
+
+```
+$ USE_SSH=true yarn deploy [or] npm run serve
```

Comment:
can we be break that into two commendas with one for each?

Reply:
changes done